users/views.py

```diff
@@ -44,28 +47,39 @@ class LoginView(APIView):
             '202': set_example(responses.login_202),
             '400': set_example(responses.login_400),
             '401': set_example(responses.login_401),
+            '404': set_example(responses.login_404)
         },
     )
+
+
+
     def post(self, request):
         serializer = LoginSerializer(data=request.data)
 
+        found_email = request.data['email']
+  
+        
         if serializer.is_valid():
             user = authenticate(
                 username=serializer.data['email'],
                 password=serializer.data['password']
-            )
-
+                )     
             if user:
                 token, _ = Token.objects.get_or_create(user=user)
-
-                return Response({
-                    'token': f"Token {token.key}"
-                }, status.HTTP_202_ACCEPTED)
+                return Response({'token': f"Token {token.key}"}, status.HTTP_202_ACCEPTED)
             else:
-                return Response({'message': 'Credentials did not match'}, status.HTTP_401_UNAUTHORIZED)
+                try:
+                    if User.objects.get(email=found_email):
+                        return Response({'message': 'Credentials did not match'}, status.HTTP_401_UNAUTHORIZED)
+                    
+                except User.DoesNotExist:
+                    raise Http404     
```

Comment:
```suggestion
                   return Response({"message": "User not found"}, status.HTTP_404_NOT_FOUND) 
```

The swagger docs output and your actual output should match.

Reply:
okay sir